build.msbuild

```diff
@@ -13,5 +16,19 @@
         <Exec Command="npm version --no-git-tag-version --allow-same-version $(PackageVersion)" />
         <Exec Command="gulp" />
         <Exec Command="nuget pack $(NuspecPath) -Version $(PackageVersion)" />
+
+        <MSBuild Projects ="$(MSBuildProjectFullPath)"
+            Properties="PackageVersion=$(PackageVersion)"
+            Targets="Test" />
+    </Target>
+
+    <Target Name="Test">
+        <Message Text="Testing Webpack build..." />
+        <MakeDir Directories="$(NpmPackOutputDir)" />
+        <Exec Command="npm pack $(MSBuildThisFileDirectory)" WorkingDirectory="$(NpmPackOutputDir)" />
```

Comment:
Do we have some way of describing that this repo needs NPM in order to build? cc @natemcmaster

Reply:
This repo isn't using korebuild, so not really.

Reply:
Ah right. I kept thinking this is the templating repo. But it is not.